dwh/validate_indicators.sh

```diff
@@ -64,5 +64,19 @@ if [[ "${counts}" != "34,13,0,4" ]]; then
     "expected to be '34,13,0,4' GOT ${counts}"
   exit 1
 fi
+
+PVLS_ratio=$(cat ${TEMP_OUT} | awk -F, '
```

Comment:
We should merge this block with the previous one but we can do it in a future PR when we add more checks here.
Also, since we are comparing floating points, we should add some error tolerance.
```suggestion
# TODO merge this awk with the previous one and add error tolerance for floating points.
PVLS_ratio=$(cat ${TEMP_OUT} | awk -F, '
```